### DISCUSSION TITLE
Call to undefined method App\Models\Tarif::setLocale()

### DISCUSSION THREAD

**Discussion by Ngouabira** [filamentphp/filament#2417] [Help]
C:\xampp\htdocs\good-sender\vendor\laravel\framework\src\Illuminate\Support\Traits\ForwardsCalls
.php
 
: 71

     *

     * @throws \BadMethodCallException

     */

    protected function forwardDecoratedCallTo($object, $method, $parameters)

    {

        $result = $this->forwardCallTo($object, $method, $parameters);

        if ($result === $object) {

            return $this;

        }

        return $result;

    }

    /**

     * Throw a bad method call exception for the given method.

     *

     * @param  string  $method

     * @return void

     *

     * @throws \BadMethodCallException

     */

    protected static function throwBadMethodCallException($method)

    {

        throw new BadMethodCallException(sprintf(

            'Call to undefined method %s::%s()', static::class, $method

        ));

    }

}

**Comment by alva-v** [comment]
I guess you are using the `Filament\Resources\Concerns\Translatable` trait on your resource. Does the corresponding model have the `Spatie\Translatable\HasTranslations` trait and a public `$translatable` array property?
